.gitattributes

```diff
@@ -0,0 +1 @@
+*.har binary
```

Comment:
What does this do?

Reply:
To avoid line end handling by git (CRLF vs LF). When running the e2e testing on windows, the line handling on har files messed up with the tests. So we are just telling git to not change the har files here

Reply:
Can you please add this as a comment to the file?